vendor/github.com/studio-b12/gowebdav/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014, Studio B12 GmbH
```

Comment:
is this a normal license in go world?
Doesn't sound too compatible with ours?

Reply:
Looks like a BSD 3-clause. It's more permissive than the Apache-2.0 license being used for this library.